### PR TITLE
Add S3 bucket and IAM user for support-api

### PR DESCRIPTION
**Pull request by rubenarakelyan**
This commit adds a new project which creates an S3 bucket and IAM user for support-api. This bucket will be used to store CSV files generated by support-api in reponse to requests from the support app.

### CODE REVIEW

File: terraform/projects/infra-support-api/main.tf [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+resource "aws_iam_policy_attachment" "s3_writer" {
+  name       = "archive-writer-policy-attachment"
+  users      = ["${aws_iam_user.support_api_app.name}"]
+  policy_arn = "${aws_iam_policy.s3_writer.arn}"
+}
+
+data "template_file" "s3_writer_policy_template" {
+  template = "${file("${path.module}/../../policies/support_api_s3_writer_policy.tpl")}"
```
Comment (katstevens) [inline terraform/projects/infra-support-api/main.tf]: Does this policy file already exist, or is it generated somewhere?

Comment (rubenarakelyan) [reply terraform/projects/infra-support-api/main.tf]: My bad, I forgot to copy it across!

File: terraform/policies/support_api_s3_writer_policy.tpl [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+             ],
+            "Resource": [
+                "arn:aws:s3:::*"
+            ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+                "s3:ListBucket",
```
Comment (barrucadu) [inline terraform/policies/support_api_s3_writer_policy.tpl]: Is `s3:ListBucket` needed here?  Isn't this covered by the section above?

File: terraform/projects/infra-support-api/main.tf [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+provider "aws" {
+  region  = "${var.aws_region}"
+  version = "1.40.0"
+}
+
+resource "aws_s3_bucket" "support_api_csvs" {
+  bucket = "govuk-${var.aws_environment}-support-api-csvs"
+
+  tags {
+    Name            = "govuk-${var.aws_environment}-support-api-csvs"
```
Comment (barrucadu) [inline terraform/projects/infra-support-api/main.tf]: Should this be capitalised?

File: terraform/policies/support_api_s3_writer_policy.tpl (hunk 2 above)
Comment (rubenarakelyan) [reply terraform/policies/support_api_s3_writer_policy.tpl]: I think so. I've removed it from here but may need to add it back in.

File: terraform/projects/infra-support-api/main.tf (hunk 3 above)
Comment (rubenarakelyan) [reply terraform/projects/infra-support-api/main.tf]: Nope, copy and paste from someone else's code!

**Pull request merged**